core/build.gradle

```diff
@@ -37,26 +36,18 @@ android {
     }
 }
 
-jacocoAndroidUnitTestReport {
-    excludes = defaultExcludes + ['**/*_Factory.class']
-}
-
-coveralls {
-    jacocoReportPath = "${buildDir}/reports/jacoco/jacocoTestReleaseUnitTestReport/jacocoTestReleaseUnitTestReport.xml"
-}
-
 buildscript {
     ext.androidx_appcompat_version = '1.1.0'
-    ext.dagger_version = '2.25.3'
     ext.powermock_version = '1.7.0'
     ext.junit_version = '4.12'
     ext.robolectric_version = '4.3.1'
     ext.androidx_test_core_version = '1.1.0'
     ext.mockito_inline_version = '2.13.0'
     ext.assertj_version = '3.9.0'
     ext.android_ktx_version = '1.1.0'
-    ext.coroutines_version = '1.3.3'
+    ext.coroutines_version = '1.4.3'
     ext.androidx_material_version = '1.2.0-alpha03'
```

Comment:
I think we can use a non alpha version for androidx.material ?

Reply:
Sure. I will bump it in another commit.

Reply:
Actually, we should bump every dependency here to its latest stable version.